### ISSUE TITLE
Crash when initializing Dexter 3.0.0 with a non-activity context

### ISSUE THREAD

**Issue by 6bangs** [Karumi/Dexter#76]
[The transparent activity is no longer started with FLAG_ACTIVITY_NEW_TASK](https://github.com/Karumi/Dexter/blob/206357e284bec2013c7dcc20372243a39625a545/dexter/src/main/java/com/karumi/dexter/DexterInstance.java#L215), which results in the following crash:

```
android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
    at android.app.ContextImpl.startActivity(ContextImpl.java:672)
    at android.app.ContextImpl.startActivity(ContextImpl.java:659)
    at android.content.ContextWrapper.startActivity(ContextWrapper.java:331)
    at com.karumi.dexter.DexterInstance.startTransparentActivityIfNeeded(DexterInstance.java:216)
    at com.karumi.dexter.DexterInstance.checkMultiplePermissions(DexterInstance.java:322)
    at com.karumi.dexter.DexterInstance.checkSinglePermission(DexterInstance.java:294)
    at com.karumi.dexter.DexterInstance.checkPermission(DexterInstance.java:80)
    at com.karumi.dexter.Dexter.checkPermission(Dexter.java:90)
```

I know 3.0.0 isn't released yet; just wanted to bring it to your attention in case it wasn't known.

**Comment by pedrovgs** [comment]
Hi @6bangs thanks for the feedback!!

I completely forgot that we've deprecated the method to initialize Dexter using the application context. I'm going to fix it before the release. 

Thanks!

**Comment by pedrovgs** [comment]
Already in review :) Thanks for your feedbak!

**Comment by 6bangs** [comment]
Awesome!

**Comment by Serchinastico** [comment]
This should be fixed in v3.0.0

Thanks for reporting and fixing it that fast guys! You rock
